types/eslint/v8/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/eslint",
-    "version": "8.56.9999",
+    "version": "8.57.9999",
```

Comment:
Same as above. Updating the version is not required.

Reply:
I’ve applied the same fix as above.